### DISCUSSION TITLE
Save page daily

### DISCUSSION THREAD

**Discussion by tripleo1** [gildas-lormeau/SingleFile#890] [Q&A]
Is it possible to save a page (or two) once in a 24 hour period after it is processed by all the ad blockers?

I hope my question/request is clear.

**Comment by gildas-lormeau** [comment]
Here is how you could achieve this:
- Set the option `File name > template` to `{page-title} ({date-locale}).html`
- Set the option `File name > file name conflict resolution` to `skip duplicate files`
- Follow this procedure to define which sites you want to save: https://github.com/gildas-lormeau/SingleFile/discussions/719
